src/katsdpk8spoc/sdpcontroller.py

```diff
@@ -116,6 +120,14 @@ async def status(self, subarray):
     async def check(self):
         pass
 
+    async def get_antennas(self):
+        """Get antennas/receptors that this controller is configured with."""
+        return self.config.get("antennas", [])
```

Comment:
I think there isn't really a case for this to _not_ be in the configuration file. Maybe raising this error is a better approach?

Reply:
Is this the maximum number of antennas? Since one of the runtime configs for a subarray (product controller) is the number of receptors* a number anywhere from 1 to 64 or soon 1 to 84 - thus we cannot have number of receptors per subarray in the config, but we should have max_number_of_receptors in the config.

* receptors are the correct term. But most people talk incorrectly of antennas.

Reply:
@martinslabber , these are the receptors that are available to be selected:
```
antennas:
   - "m000"
   - "m001"
   - "m002"
   - "m003"
```

Reply:
so the operator chooses specific receptors as is currently the case.